packages/neon/neon/lib/src/app.dart

```diff
@@ -284,42 +285,53 @@ class _NeonAppState extends State<NeonApp> with WidgetsBindingObserver, tray.Tra
   }
 
   @override
-  Widget build(final BuildContext context) => OptionsCollectionBuilder(
-        valueListenable: _globalOptions,
-        builder: (final context, final options, final _) => StreamBuilder<Account?>(
-          stream: _accountsBloc.activeAccount,
-          builder: (final context, final activeAccountSnapshot) {
-            FlutterNativeSplash.remove();
-            return ResultBuilder<core.OcsGetCapabilitiesResponseApplicationJson_Ocs_Data?>.behaviorSubject(
-              subject: activeAccountSnapshot.hasData
-                  ? _accountsBloc.getCapabilitiesBlocFor(activeAccountSnapshot.data!).capabilities
-                  : null,
-              builder: (final context, final capabilitiesSnapshot) {
-                final appTheme = AppTheme(
-                  capabilitiesSnapshot.data?.capabilities.themingPublicCapabilities?.theming,
-                  keepOriginalAccentColor: options.themeKeepOriginalAccentColor.value,
-                  oledAsDark: options.themeOLEDAsDark.value,
-                  appThemes: _appImplementations.map((final a) => a.theme).whereNotNull(),
-                  neonTheme: widget.neonTheme,
-                );
-
-                return MaterialApp.router(
-                  localizationsDelegates: [
-                    ..._appImplementations.map((final app) => app.localizationsDelegate),
-                    ...NeonLocalizations.localizationsDelegates,
-                  ],
-                  supportedLocales: {
-                    ..._appImplementations.map((final app) => app.supportedLocales).expand((final element) => element),
-                    ...NeonLocalizations.supportedLocales,
+  Widget build(final BuildContext context) => DynamicColorBuilder(
+        builder: (final deviceThemeLight, final deviceThemeDark) {
+          if (deviceThemeLight == null || deviceThemeDark == null) {
+            return const SizedBox.shrink();
+          }
```

Comment:
Having null is a perfectly fine value:
>  /// Builds the child widget of this widget, providing a light and dark [ColorScheme].
  ///
  /// The [ColorScheme]s will be null if dynamic color is not supported on the
  /// platform, or if the OS has yet to respond.
  
  The corresponding plugin calls (`DynamicColorPlugin.getCorePalette` and `DynamicColorPlugin.getAccentColor`) return null on unsupported devices. This would mean we'd never display the app for them.
  As we construct our own theme and colorscheme anyways we don't need separate dark and light colorschemes. I'd rather call these methods ourselves in a future builder where we'd also get the connection state and build our colorscheme from the provided one/provided accent color.

Reply:
Ah I thought null would only happen while the device theme is still loading.
The problem is getting the device theme is async so we need some kind builder widget to handle it.
I don't understand how you would drop one of the color schemes since we need both to build the theme?
It would be better for us if not the color scheme but the raw color palette was exposed.

Reply:
>  It would be better for us if not the color scheme but the raw color palette was exposed.

Abd that's exactly what the raw methods do. I encurage you to look at the implementation of the builder and you'll find it obvious:
https://github.com/material-foundation/flutter-packages/blob/main/packages/dynamic_color/lib/src/dynamic_color_builder.dart

Reply:
I fixed the null thing, but is there something that needs to be changed about the two color schemes?

Reply:
Well I looked at the implementation of the builder and decided it is easier to use it instead of copying the logic into our own widget.

Reply:
Not directly.

Reply:
And not using it would still require to variables because we need both the colorpalette and the accent color.

Reply:
So can we just stick to this?

Reply:
Let's go with this for now.